src/cascadia/TerminalControl/TermControl.idl

```diff
@@ -7,6 +7,14 @@ namespace Microsoft.Terminal.TerminalControl
     delegate void FontSizeChangedEventArgs(Int32 width, Int32 height, Boolean isInitialChange);
     delegate void ScrollPositionChangedEventArgs(Int32 viewTop, Int32 viewHeight, Int32 bufferLength);
 
+    // c++/winrt makes it difficult to share this idl between two projects,

```

Comment:
Is there any reason we don't just define this in the `TerminalControl` project, and just reference it as `Microsoft.TerminalControl.IF7Listener` in `TerminalApp`?

Reply:
I .. honestly don't know why I didn't choose that. It seems wrong for WindowsTerminal.exe to be depending on an interface from all the way down in TerminalControl; it's an app concern, and Control just happens to share the same concern.